pipelines/common/src/main/java/com/google/fhir/analytics/ParquetUtil.java

```diff
@@ -41,6 +41,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+// Implementing AutoCloseable does not help much in the usage pattern we have with DoFns.
```

Comment:
nit: Can you please add more context here, I was not able to understand the full intent of this.

Reply:
Done.